docs/sdk/server-side-sdks/node.md

```diff
@@ -110,3 +110,32 @@ You can fetch all segmented features for a user:
 const features = dvcClient.allFeatures(user)
 ```
 See [getFeatures](https://docs.devcycle.com/bucketing-api/#operation/getFeatures) on the Bucketing API for the feature response format.
+
+### EdgeDB
+
+EdgeDB allows you to save user data to our EdgeDB storage so that you don't have to pass in all the user data every time you identify a user. Read more about [EdgeDB](https://docs.devcycle.com/docs/home/feature-management/edgedb/).
```

Comment:
need to make this clear that EdgeDB is ONLY available with Cloud Bucketing and won't have any impact for Local Bucketing